### PR TITLE
Create ubuntu.md

### PR DESCRIPTION
**Pull request by drgorb**
in order to give others a head start, I have written down the steps I had to take to successfully build MoneyChanger

### CODE REVIEW

File: doc/ubuntu.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Build from Source on Ubuntu
+
+This document has been written for ubuntu 15.10 but it should work on debian and give a good indication for other systems
+
+## Open Transactions Library
+
+You must have the [open-transactions library installed](https://github.com/Open-Transactions/opentxs-notary/blob/develop/docs/INSTALL-Debian_Ubuntu.txt)
+
+It must be the same branch as MoneyChanger: if you are building the `develop` branch f the MoneyChaner, that is the branch you want to checkout for the Open Transactions library as well
```
Comment (justusranvier) [inline doc/ubuntu.md]: Have a couple spelling errors in this line

File: doc/ubuntu.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+moneychanger-/moneychanger-qt` 
+## Building MoneyChanger
+
+You can try to build the `master` branch first. If that does not succeed, try the `develop` branch instead.
```
Comment (justusranvier) [inline doc/ubuntu.md]: Nobody should build the master branch. It's outdated and useless. Really, we ought to remove it from the repository entirely...

Comment (FellowTraveler) [inline doc/ubuntu.md]: I just merged master and develop, so hopefully will cut down on some confusion. You're right though, we should dump master branch for now.

**Comment by yamamushi** [comment]
Can you pull this into the develop branch? We don't pull into Master as a rule of thumb.

**Comment by FellowTraveler** [comment]
drgorb, I am closing this PR since we needed it to be submitted to the develop branch, not the master branch.

HOWEVER, I went ahead and copied your ubuntu.md file myself into the docs folder of the develop branch! So it's still been "merged" so to speak. Thanks again for your great contribution!

**Pull request closed without merging**